tests/test_model_type.py

```diff
@@ -182,3 +182,25 @@ class M(Model):
     assert M.to_one.model_class is M
     assert M.to_many.field.model_class is M
     assert M.matrix.field.field.model_class is M
+
+
+def test_model_context_pass_to_type():
+    from schematics.types import BaseType
+
+    class CustomType(BaseType):
+
+        def to_native(self, value, context=None):
+            suffix = context.suffix
+            return str(value) + suffix
+
+        def to_primitive(self, value, context=None):
+            suffix = context.suffix
+            return value[:-len(suffix)]
+
+    class Thing(Model):
+        x = CustomType()
+
+    context = {'suffix': 'z'}
```

Comment:
Hi @bintoro note that I'm passing a dict here for the context. An actual Context instance didn't work out well, which surprised me.